### DISCUSSION TITLE
小调查 | 截止至目前，使用网页版 ChatGPT，并且是通过浏览器登录模式成功的同学有哪些？

### DISCUSSION THREAD

**Discussion by lss233** [lss233/chatgpt-mirai-qq-bot#147] [Polls]
根据 #143，现在似乎很多人遇到了浏览器登录成功但没有退出，造成卡死的情况。  

我测试了一下， 发现是程序想要获得一个叫 `_puid` 的 Cookie，但网页并没有提供，所以导致了卡死。 

但无浏览器模式因为请求是在第三方服务器进行的，所以绕过了这个问题。

**Comment by joe12801** [comment]
我用新手 docker 无浏览器模式

**Reply by jeffreys125** [reply]
我开始用browser模式一直卡Found session token，最后只能在本机开个代理，用proxy模式

**Comment by leetingo** [comment]
我这边1.5.4版本没有遇到“浏览器登录成功但没有退出”问题，在1.5.4后的版本都存在这个问题

**Reply by laoutiao** [reply]
是的 我也一样 我用1.5.4都可以用 但往上其他版本都是登陆成功，但是xpra没有获取到

**Reply by lss233** [reply]
我现在也是这样，所以就觉得蛮奇怪的
